aws-lc-rs/README.md

```diff
@@ -70,6 +70,30 @@ the pre-generated bindings. This feature requires `libclang` to be installed. Se
 [requirements](https://rust-lang.github.io/rust-bindgen/requirements.html)
 for [rust-bindgen](https://github.com/rust-lang/rust-bindgen)
 
+##### prebuilt-nasm
```

Comment:
nit:  Should the flag be use-prebuilt-nasm?

Reply:
I think a more accurate name would be "allow-prebuilt-nasm", since we will still use the NASM assembler if found. I prefer the shorter "prebuilt-nasm", but I could likely be convinced otherwise.

Reply:
sounds good to me